index.bs

```diff
@@ -139,6 +139,9 @@ The printer operation is implementation-defined. It accepts a log level indicati
 
 By the time the printer operation is called, all format specifiers will have been taken into account, and any arguments that are meant to be consumed by format specifiers will not be present in <var>args</var>. The implementation's job is simply to print the List.
 
+If the console is not open, implementations should choose to buffer messages
```

Comment:
> not open "when the printer operation is called".

great this more precise, <3 it

> Maybe add something like ", up to an implementation-chosen limit (typically on the order of at least 100)".
 
good call... wanted to write something about the limits but had a hard time finding the right language... fixing...